app/controllers/medicaid/health_pregnancy_controller.rb

```diff
@@ -1,12 +1,20 @@
 module Medicaid
   class HealthPregnancyController < MedicaidStepsController
+    def edit
+      super
+
+      @step.members = current_application.members
```

Comment:
why add to `update` rather than define `step` here like you did in `ContactSocialSecurityController`? Feels like they are similar scenarios?

Reply:
The flow is _ever so slightly_ different for steps that inherit from the ManyMembersStepsController and the MedicaidStepsController. Because of that, the behavior is a lil bit different.

But you know me, I'd love to tackle making that the same as well. :)